backend/src/test/java/com/carffeine/carffeine/station/controller/station/StationControllerTest.java

```diff
@@ -154,4 +158,57 @@ class StationControllerTest {
                         )
                 ));
     }
+
+    @Test
+    void 충전소를_검색한다() throws Exception {
+        when(stationService.searchStations(any(), any(), anyInt()))
+                .thenReturn(new StationsSearchResponse(
+                        2,
+                        List.of(
+                                new StationSearchResponse(
+                                        "stationId",
+                                        "stationName",
+                                        List.of(
+                                                "QUICK",
+                                                "STANDARD"
+                                        ),
+                                        "address",
+                                        BigDecimal.valueOf(37.123456),
+                                        BigDecimal.valueOf(127.123456)
+                                ),
+                                new StationSearchResponse(
+                                        "stationId2",
+                                        "stationName2",
+                                        List.of(
+                                                "QUICK"
+                                        ),
+                                        "address2",
+                                        BigDecimal.valueOf(37.123456),
+                                        BigDecimal.valueOf(127.123456)
+                                )
+                        )
+                ));
+
+        mockMvc.perform(RestDocumentationRequestBuilders.get("/api/stations/search?q=stationName&page=1&scope=stationName&scope=address&scope=speed&scope=latitude&scope=longitude"))
```

Comment:
```suggestion
        mockMvc.perform(RestDocumentationRequestBuilders.get("/api/stations/search?q=선릉역 충전소&page=1&scope=stationName&scope=address&scope=speed&scope=latitude&scope=longitude"))
```
이런식으로 하는 것이 가독성이 좋을 것 같아요

Reply:
알겠습니다 👍